core/schema/container.graphqls

```diff
@@ -63,6 +63,9 @@ type Container {
 
     "Target build stage to build."
     target: String
+
+    "Secrets to pass to the build."
```

Comment:
Maybe add a small sentence about the `/run/secrets` so the user knows where the secret will be from the doc.

Reply:
Very good point, thanks.